float64[6] lateral_force

float64[6] normal_force